files/galaxy/tpv/tools.yml.j2

```
@@ -890,3 +886,9 @@ tools:
     scheduling:
       require:
         - singularity
+
+  ".*srma_wrapper.*":
+    scheduling:
+      require:
+        - singularity
+    singularity_default_container_id: "{{ cvmfs.singularity.path }}/all/python:2.7.16"  
```

```suggestion
    params:
      singularity_default_container_id: "{{ cvmfs.singularity.path }}/all/python:2.7.16"  
```

[Spotted](https://github.com/usegalaxy-eu/infrastructure-playbook/actions/runs/5462600859/jobs/9942178140?pr=830#step:21:31) in dry-run.